environments.json

```diff
@@ -157,11 +157,11 @@
     "family": "Chakra",
     "short": "IE 8",
     "release": "2009-03-19",
+    "retire": "2016-01-12",
     "obsolete": true,
     "test_suites": [
       "es5",
-      "es6",
-      "esnext",
+      "esintl",
```

Comment:
(same logic as above; if IE 9 is zeroed and displayed, then it makes sense to hide IE 8)